models/product-type/package.json

```diff
@@ -8,20 +8,30 @@
     "url": "https://github.com/commercetools/test-data.git",
     "directory": "models/product-type"
   },
-  "keywords": ["javascript", "typescript", "test-data"],
+  "keywords": [
+    "javascript",
+    "typescript",
+    "test-data"
+  ],
   "license": "MIT",
   "publishConfig": {
     "access": "public"
   },
   "main": "dist/commercetools-test-data-product-type.cjs.js",
   "module": "dist/commercetools-test-data-product-type.esm.js",
-  "files": ["dist", "package.json", "LICENSE", "README.md"],
+  "files": [
+    "dist",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
   "dependencies": {
     "@babel/runtime": "^7.17.9",
     "@babel/runtime-corejs3": "^7.17.9",
     "@commercetools-test-data/commons": "10.2.0",
     "@commercetools-test-data/core": "10.2.0",
     "@commercetools-test-data/utils": "10.2.0",
+    "@commercetools-test-data/type": "10.1.1",
```

Comment:
Just a reminder to update the version of this package, in-sync with other packages before merging this PR. The latest version is `10.3.0`